### PR TITLE
Changed bash read for readarray funtion

### PR DESCRIPTION
**Pull request by lgallard**
I had to to change the read function for the readarray function in order to make it work.

### CODE REVIEW

File: static-router-config [hunk 1]
```diff
@@ -20,11 +20,12 @@ then
   exit 1
 fi
 
-AWS_ADDRESSES_TO_USE=$(grep --color -A2 'Address' $1 | sed 's/.*:\ \(.*\)$/\1/' | grep '^[0-9].*$' | sed 's/^\(.*\)\/.*$/\1/')
-read -a AWS_ADDRESSES_TO_USE <<< $AWS_ADDRESSES_TO_USE
+AWS_ADDRESSES_TO_USEt=$(grep --color -A2 'Address' $1 | sed 's/.*:\ \(.*\)$/\1/' | grep '^[0-9].*$' | sed 's/^\(.*\)\/.*$/\1/')
+readarray -t AWS_ADDRESSES_TO_USE <<< $AWS_ADDRESSES_TO_USEt
```
Comment (matelang) [inline static-router-config]: I think you have a typo here `$AWS_ADDRESSES_TO_USEt`

What is the advantage of using read vs readarray by the way?

Comment (lgallard) [reply static-router-config]: Typo fixed. 

Regarding _read_ -a vs _readarray_ [1], the only difference I see is that _read_ overwrites any previous data. I think in some bash versions it doesn't work as it used to. In my case your code doesn't run on GNU bash, version 4.4.11(1)-release (x86_64-pc-linux-gnu), that's why I had to changed it and now it works like a charm.

[1] https://www.gnu.org/software/bash/manual/bashref.html#Arrays

**Comment by matelang** [comment]
@lgallard can you please specify what configuration you have?
BaSH version, `uname -a`, eventually `lsb_release -a`?

**Comment by lgallard** [comment]
Sure, no problem:

$ bash --version
GNU bash, version 4.4.12(1)-release (x86_64-pc-linux-gnu)
Copyright (C) 2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html
>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

$ uname -a
Linux lenovo 4.9.0-2-amd64 #1 SMP Debian 4.9.18-1 (2017-03-30) x86_64
GNU/Linux

$ lsb_release -a
No LSB modules are available.
Distributor ID: Debian
Description: Debian GNU/Linux 9.0 (stretch)
Release: 9.0



Luis Gallardo
http://lgallardo.com

**Review by matelang — APPROVED** [review]

**Pull request merged**